src/feature.rs

```diff
@@ -65,24 +65,28 @@ pub fn filterbanks(
 
     // we should convert Mels back to Hertz because the start and end-points
     // should be at the desired frequencies.
-    let hertz = mel_to_frequency(mels);
 
     // The frequency resolution required to put filters at the
     // exact points calculated above should be extracted.
     //  So we should round those frequencies to the closest FFT bin.
-    let freq_index = ((coefficients as i32 + 1) * hertz / sampling_freq).floor();
+    let freq_index = mel_to_frequency(mels)
+        .map(|x| ((coefficients as i32 + 1) as f64 * x / sampling_freq) as usize);
```

Comment:
I'm no entirely too sure, but I think that keeping this is as `(coefficients as i32 + 1) as f64 * mel_to_frequency(mels) / sampling_freq` would be faster as we could be profitting from SIMD operations. However, we would need to remember that what we have are floats and do the transformation later.